packages/plugin-msdynamic-api/src/graphql/resolvers/syncLog.ts

```
@@ -15,13 +15,19 @@ export default {
       subdomain,
       action: "users.findOne",
       data: { _id: syncLog.createdBy },
-      isRPC: true
+      isRPC: true,
     });
   },
 
   async content(syncLog: ISyncLogDocument, {}, {}: IContext) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)